### PR TITLE
Check for conflicting binarisation

### PR DESCRIPTION
**Pull request by johenglisch**
Work on feature checks:

 * Add unit tests for all the existing feature checks.
 * Fix a few checks where I checked for the *integer* 0 instead of the *string* ‘0’.
 * Check if binarised features match up.
 * Complain when binarised *ternary* features are both 0.

### CODE REVIEW

File: src/pygrambank/sheet.py [hunk 1]
```diff
@@ -60,20 +60,20 @@ def _require_comment(feat, reason):
         == '0'
     ):
         reason = 'GB333, GB334, GB335, and GB336 are all 0'
-        for feat in ('GB333', 'GB334', 'GB335', 'GB336'):  # pragma: nocover
```
Comment (HedvigS) [inline src/pygrambank/sheet.py:63]: what did "pragma: no cover" mean?

Comment (johenglisch) [reply src/pygrambank/sheet.py:63]: The test suite creates a coverage report at the end that says which lines of code were run during testing and which were not.  This comment just tells the test suite ‘no need to report that line -- we have our reason for why we don't test it’ (although in this case I don't remember what my reason actually was back then (<\_<)" ).

Now the lines *are* being tested properly so I removed the comment.

Comment (HedvigS) [reply src/pygrambank/sheet.py:63]: Ah, makes sense :)

**Review by HedvigS — APPROVED** [review]

**Comment by HedvigS** [comment]
are we waiting for @xrotwang to review too?

**Review by xrotwang — APPROVED** [review]

**Comment by xrotwang** [comment]
> are we waiting for @xrotwang to review too?

I think I'm no longer on top of things in detail when it comes to pygrambank.

**Pull request merged**